cmake/pkgconfig/ignition-component.pc.in

```diff
@@ -2,8 +2,8 @@ prefix=${pcfiledir}/@PC_CONFIG_RELATIVE_PATH_TO_PREFIX@
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/@IGN_INCLUDE_INSTALL_DIR_POSTFIX@
 
-Name: Ignition @IGN_DESIGNATION@ @component_name@
-Description: A set of @IGN_DESIGNATION@ @component_name@ classes for robot applications
+Name: Gazebo @GZ_DESIGNATION@ @component_name@
```

Comment:
This will be weird for `ign-gazebo`, because the designation will still be "Gazebo", so this becomes `Gazebo Gazebo`